exercises/isbn-verifier/src/test/kotlin/IsbnVerifierTest.kt

```diff
@@ -4,100 +4,74 @@ import kotlin.test.assertFalse
 import kotlin.test.assertTrue
 
 class IsbnVerifierTest {
-    private var isbnVerifier: IsbnVerifier = IsbnVerifier()
+    @Test
+    fun `valid | reguar isbn`() = assertValid("3-598-21508-8")
 
+    @Ignore
     @Test
-    fun validIsbnNumber() {
-        assertTrue(isbnVerifier.isValid("3-598-21508-8"))
-    }
+    fun `invalid | check digit`() = assertInvalid("3-598-21508-9")
 
     @Ignore
     @Test
-    fun invalidIsbnCheckDigit() {
-        assertFalse(isbnVerifier.isValid("3-598-21508-9"))
-    }
+    fun `valid | check digit is 10`() = assertValid("3-598-21507-X")
 
     @Ignore
     @Test
-    fun validIsbnNumberWithCheckDigitOfTen() {
-        assertTrue(isbnVerifier.isValid("3-598-21507-X"))
-    }
+    fun `invalid | check digit is character other than X`() = assertInvalid("3-598-21507-A")
 
     @Ignore
     @Test
-    fun checkDigitIsACharacterOtherThanX() {
-        assertFalse(isbnVerifier.isValid("3-598-21507-A"))
-    }
+    fun `invalid | forbidden character`() = assertInvalid("3-598-P1581-X")
```

Comment:
Should we stick to the description in the canonical data? It is "invalid character in isbn"

Reply:
`invalid` is already used for the *case* prefix, so I just failed to find proepr synonym. Do you have a suggestion?

Reply:
Aha I see. No I guess it's fine then